### ISSUE TITLE
Installation fails with "mkdir -p app/lib" on Windows 7

### ISSUE THREAD

**Issue by MatteoGgl** [jiahaog/nativefier#57]
After trying to install with `npm install nativefier -g` I get the following output:

```
npm ERR! Windows_NT 6.1.7601
npm ERR! argv "C:\\Program Files\\nodejs\\node.exe" "C:\\Program Files\\nodejs\\node_modules\\npm\\bin\\npm-cli.js" "install" "nativefier" "-g"
npm ERR! node v4.2.2
npm ERR! npm  v2.14.7
npm ERR! code ELIFECYCLE

npm ERR! nativefier@6.5.5 postinstall: `mkdir -p app/lib`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the nativefier@6.5.5 postinstall script 'mkdir -p app/lib'.
npm ERR! This is most likely a problem with the nativefier package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     mkdir -p app/lib
npm ERR! You can get their info via:
npm ERR!     npm owner ls nativefier
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     C:\Program Files\Git\npm-debug.log
```
And here's the [npm-debug.log](https://github.com/jiahaog/nativefier/files/100913/npm-debug.txt) file, if it's of any help.

**Comment by jiahaog** [comment]
Sorry about that, I overlooked the issue with windows `mkdir -p`, and it's fixed in `v6.5.6`. Do let me know if the problem persists!

**Comment by MatteoGgl** [comment]
Everything seems to work fine now! Thanks for the fix!
